proto/BUILD.bazel

```diff
@@ -3,6 +3,7 @@ package(
     licenses = ["notice"],  # Apache 2.0
 )
 
+load("@bazel_skylib//rules:build_test.bzl", "build_test")
```

Comment:
I don't see `bazel_skylib` as a dependency anywhere in https://github.com/p4lang/p4runtime/blob/main/proto/p4runtime_deps.bzl?